jetty-distribution/src/main/resources/etc/fileserver.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure.dtd">
+
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+
+  <Set name="handler">
+    <New class="org.eclipse.jetty.server.handler.HandlerList">
+      <Set name="handlers">
+        <Array type="org.eclipse.jetty.server.Handler">
+          <Item>
+            <New class="org.eclipse.jetty.server.handler.ResourceHandler">
+              <Set name="directoriesListed">true</Set>
+              <Set name="welcomeFiles">
+                <Array type="String"><Item>index.html</Item></Array>
+              </Set>
+              <Set name="resourceBase"><Property name="jetty.base" default="."/>/docs/</Set>
+            </New>
+          </Item>
+          <Item>
+            <New class="org.eclipse.jetty.server.handler.DefaultHandler">
+            </New>
+          </Item>
+        </Array>
+      </Set>
+    </New>
+  </Set>
+
+  <Call name="insertHandler">
```

Comment:
module should not create it's own request log, that is provided by another module.